.travis.yml

```diff
@@ -21,6 +21,11 @@ cache:
 before_install:
   - npm install -g npm@5.5.1
   - sudo apt-get install libgconf2-4
+  - export SOURCE_BRANCH=${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH}
+  - export IS_FORKED_PULL_REQUEST=$([[ $TRAVIS_PULL_REQUEST != false && $TRAVIS_PULL_REQUEST_SLUG != $TRAVIS_REPO_SLUG ]] && echo true || echo false)
+  - export IS_BETA=$([[ $TRAVIS_TAG =~ .*beta.* && $IS_FORKED_PULL_REQUEST = false ]] && echo true || echo false)
+  - export IS_RELEASE=$([[ $SOURCE_BRANCH =~ ^.*release.*$|^master$ && $IS_FORKED_PULL_REQUEST = false ]] && echo true || echo false)
+  - export IS_FEATURE=$([[ $SOURCE_BRANCH =~ ^JSUI-[0-9]+$ && $IS_FORKED_PULL_REQUEST = false ]] && echo true || echo false)
```

Comment:
These variables are introduced to improve the readability of the conditions.

They contain `"true"` and `"false"` instead of `0` and `1` to avoid ambiguity.

`IS_RELEASE` is evaluated by the `^.*release.*$|^master$` regular expression instead of the `! $TRAVIS_TAG =~ .*beta.*` regular expression since these conditions are also going to be evaluated on `JSUI-*` branches, from now on.

Values tested on this PR:
![image](https://user-images.githubusercontent.com/54454747/87351659-c0f68800-c527-11ea-90c0-03f443ccdfae.png)

Values tested on a fork's PR:
![image](https://user-images.githubusercontent.com/54454747/87351824-fef3ac00-c527-11ea-8410-c57931e9abcb.png)